src/wp-content/themes/twentyseventeen/inc/block-patterns.php

```diff
@@ -0,0 +1,198 @@
+			<div class="wp-block-image is-style-default"><figure class="aligncenter size-large"><img src="' . get_template_directory_uri() . '/assets/images/direct-light.jpg" alt="Direct Light"/></figure></div>
+			<!-- /wp:image -->
+            <!-- wp:heading {"align":"left","textColor":"black","style":{"typography":{"fontSize":30}}} -->
+            <h2 class="has-text-align-left has-black-color has-text-color" style="font-size:30px">Direct Light</h2>
+            <!-- /wp:heading -->
+            <!-- wp:paragraph {"align":"left"} -->
+            <p class="has-text-align-left"><a href="http://wordpress.org/">' . __( 'See Case Study' ) . ' →</a></p>
+            <!-- /wp:paragraph --></div></div>
+            <!-- /wp:group --></div>
+            <!-- /wp:column --></div>
+            <!-- /wp:columns -->
+            <!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->',
+		)
+	);
+
+	register_block_pattern(
+		'twentyseventeen/images-with-link',
+		array(
+			'title'      => __( 'Services', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+            
+            <!-- wp:heading {"level":1,"style":{"typography":{"fontSize":50}}} -->
+            <h1 style="font-size:50px">' . __( 'Our Services' ) . '</h1>
+            <!-- /wp:heading -->
+            
+            <!-- wp:columns -->
+            <div class="wp-block-columns"><!-- wp:column -->
+            <div class="wp-block-column">
+            <!-- wp:paragraph {"style":{"typography":{"fontSize":21, "lineHeight":"2.5"}}} -->
+            <p style="font-size:21px"><a href="http://wordpress.org/">' . __( 'Branding' ) . ' →</a><br><a href="http://wordpress.org/">' . __( 'Webdesign' ) . ' →</a><br><a href="http://wordpress.org/">' . __( 'Web Development' ) . ' →</a></p>
+            <!-- /wp:paragraph -->
+            </div>
+            <!-- /wp:column -->
+            
+            <!-- wp:column -->
+            <div class="wp-block-column">
+            <!-- wp:paragraph {"style":{"typography":{"fontSize":21, "lineHeight":"2.5"}}} -->
+            <p style="font-size:21px"><a href="http://wordpress.org/">' . __( 'Content Strategy' ) . ' →</a><br><a href="http://wordpress.org/">' . __( 'Marketing &amp; SEO' ) . ' →</a><br><a href="http://wordpress.org/">' . __( 'Video Production' ) . ' →</a></p>
+            <!-- /wp:paragraph --></div>
+            <!-- /wp:column --></div>
+            <!-- /wp:columns -->
+            
+            <!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->',
+		)
+	);
+
+	register_block_pattern(
+		'twentyseventeen/images-with-link',
```

Comment:
This pattern & the one above it are using the same pattern name as "Images with Link", so they're overriding each other — the only patterns I see in Gutenberg are "Large Heading with Button", "Images with Text & Link", and "Contact Us".